scripts/Crawlers/constants.py

```diff
@@ -0,0 +1,60 @@
+LICENSE_CODES = [
+    "CC BY-NC-SA",
+    "CC BY-NC-ND",
+    "CC BY-SA",
+    "CC BY-NC",
+    "CC BY-ND",
+    "CC BY",
+    "CC0",
+    "ODbL" "ODC-By",
+    "PDDL",
+]
+NO_ANNEX_FILE_PATTERNS = ["**/DATS.json", "**/README*", "**/LICENSE*"]
+REQUIRED_DATS_FIELDS = [
+    "title",
+    "types",
+    "creators",
+    "licenses",
+    "description",
+    "keywords",
+    "version",
+]
+DATS_FIELDS = [
+    "title",
+    "identifier",
+    "creators",
+    "description",
+    "version",
+    "licenses",
+    "keywords",
+    "distributions",
+    "extraProperties",
+    "alternateIdentifiers",
+    "relatedIdentifiers",
+    "dates",
+    "storedIn",
+    "spatialCoverage",
+    "types",
+    "availability",
+    "refinement",
+    "aggregation",
+    "privacy",
+    "dimensions",
+    "primaryPublications",
+    "citations",
+    "citationCount",
+    "producedBy",
+    "isAbout",
+    "hasPart",
+    "acknowledges",
+]
+MODALITIES = {
+    "fMRI": ["bold", "func", "cbv"],
+    "MRI": ["T1", "T2", "FLAIR", "FLASH", "PD", "angio", "anat", "mask"],
+    "diffusion": ["dwi", "dti", "sbref"],
+    "meg": ["meg"],
+    "intracranial eeg": ["ieeg"],
+    "eeg": ["eeg"],
```

Comment:
```suggestion
    "MEG": ["meg"],
    "intracranial EEG": ["ieeg"],
    "EEG": ["eeg"],
```

in theory, MEG and EEG should be all upper case as they are acronyms like fMRI and MRI